### DISCUSSION TITLE
Anomaly Published

### DISCUSSION THREAD

**Discussion by jithendra945** [cuebook/CueObserve#84] [Q&A] [answered]
How are the Anomalies are categorizing as published and non-published.

Most of them are not Published, Only some are getting Published.

May I know,
How this Published thing is working ?

**Comment by sachinkbansal** [comment, marked as answer]
Anomalies are published when the actual value is outside the forecasted value's band. i.e when deviation is greater than 0.

See https://cueobserve.cuebook.ai/anomaly-detection#anomaly-deviation for a visual representation of deviation.

**Reply by jithendra945** [reply]
the red dot is an anomaly right, when it is outside Range, Im having some graphs which contains Anomalies but are not published.

**Reply by sachinkbansal** [reply]
Yes, it's an anomaly when the red dot is outside the grey band.

Anomaly card is published if the latest anomaly is within 5 days for daily granularity or 1 day for hourly granularity. See https://cueobserve.cuebook.ai/anomaly-detection#create-card.

If your graph doesn't follow this, can you please share a screenshot of the graph. You can hide confidential data in the screenshot.
